### PR TITLE
Emagged/Hacked cyborgs can no longer be detonated / locked down

### PR DESCRIPTION
**Pull request by MLGTASTICa**
## About The Pull Request

Emagged/Hacked cyborgs can no longer be remotely detonated/locked
## Why It's Good For The Game
Being able to perma-kill any malfunctioning / emagged cyborg spontaneously and without warning is both extremly annoying and extremly unfair for the cyborgs . You can't remotely detonate a traitor , so you shouldn't be able to remotely detonate a hacked cyborg either. 
## Changelog
:cl:
balance: Hacked/Emagged cyborgs can no longer be detonated / locked down
/:cl:

### CODE REVIEW

**Comment by Echopsychotic** [comment]
Love it, I hated being detonated and just round removed rather than locked down. While I think keeping lockdown would be better but disabling the explode. or if they tried to use the explode it instead blew up the console. Now that would be a better meme.

**Comment by miniusAreas** [comment]
I oppose this. Cyborgs who are detonated shouldn't be perma-killed, but shouldn't be able to *not* be stopped remotely either. Cyborgs are very powerful tools, hence why lockdown exists. It is very annoying to be emagged or hacked, then blown up and round removed, but does not warrant removing the ability to do so fully. 

This also allows people to metagame a emagged or hacked borg way easier.

**Comment by MLGTASTICa** [comment]
Anyone can find out if a borg is hacked/emagged if they law 2 them to state laws ,  gameplay wise cyborgs   are as powerfull as their AA can get them ,they can be stunlocked with flashes/flashbangs and also hardstunned with emps/ions.
This offers hacked/emagged borgs a better chance , one-click  win buttons are overall bad for gameplay

**Comment by miniusAreas** [comment]
And any borg can fake stating another set of laws because of their emagged laws. It's still a one click button with the flash aside from beating them to fucking death, or opening their pannel and removing their cell or hacking them into lockdown. There's tons of oneclick buttons in SS13 - If your master wasn't smart enough to remove the cyborg console, that's on them.

**Comment by MLGTASTICa** [comment]
Remote detonations barely have any counter unless you are malf AI , and even then you'd have to detonate all the circuit imprinters, plasmafire tech storage , and destroy the science and engineering tech fabs, which is by itself preety impossible. Borgs can keep distance to avoid flashes / EMP's  , remote detonations have no warning.

**Comment by ThatGuyFromThatPlace** [comment]
The borg is not a traitor. The borg is an aid to the traitor, both in the case of MalfAI and emagged borgs. You shouldn't be able to effectively create All-access-wielding synth traitors without flash protection by swiping an emag.

**Comment by Echopsychotic** [comment]
> And any borg can fake stating another set of laws because of their emagged laws (it's meta to call them out on stating them slowly). It's still a one click button with the flash aside from beating them to fucking death, or opening their pannel and removing their cell or hacking them into lockdown. There's tons of oneclick buttons in SS13 - If your master wasn't smart enough to remove the cyborg console, that's on them.



At the moment, the meta behind the speed at which a borg states its laws is a problem. I witnessed this first hand as a borg that was emagged. Someone knew full well that the borg was emagged and went on the sec channel afterward and said the borgs were malf because of the late delay between states. Then at the end of the round we were all blown and it gibbed the traitor even though we only harmed people that got in the way. there was no lockdown, only detonation.
I am saying this because borgs are weak to hard stuns like sudden flash equipment and EMP. Emagged borgs should be removed from the console as they are no longer linked to the station at all in that case. So I agree with this. If you left them on the console I would say still make it possible for lockdown and if they try to detonate have there be a malfunction and blow the console itself instead. Powersurge or something.

**Comment by MLGTASTICa** [comment]
Trying to detonate a emagged/hacked borgs just tells them that the console had a error and stops the action from happening , having it blow up would be a bit too punishing

**Comment by Echopsychotic** [comment]
What if its malf AI that hacked their borgs? Powersurge since they can just blow the console anyways.

**Comment by MLGTASTICa** [comment]
Malf AI hacked borgs are also unblowable , since they get emagged code-wise

**Comment by Owai-Seek** [comment]
I agree with the remote explosion removal, but not the lockdown. There's no other feasible way to non-lethally stop borgs. Unless an alternative is produced, the remote lock-down should stay, or they're just going to get shot with lethals.

**Comment by ForrestWick** [comment]
While I'm in favor of this in theory, it means RDs and captains can just meta it super easily and will just check the buttons randomly. I can guarantee it will happen enough to be a problem

**Comment by Echopsychotic** [comment]
What if the borg was just removed from the console altogether?

**Comment by MLGTASTICa** [comment]
if the borg was removed from the console it would be easier to meta , having them still appear to function as normal until you do try to lock / detonate them is a better alternative, and it will get admin-logged , making it very easy to deal with metagaming.
Allowing the lockdown is basically a detonation in the sense that the borg cannot do anything anymore and people will still likely gun it down , ontop of the fact that the cyborg cannot have its lockdown removed other than by the remote.
Cyborgs can be non-lethally stopped wit flashbangs , flashes and EMP's.

**Comment by Echopsychotic** [comment]
> @Echopsychotic A lot of those options rely on time and resources that competent silicons can generally deny from you (if anyone is even smart enough to take advantage of it). Generally speaking, you should expect the silicons at the very minimum to take over science so they can't straight up blow borgs with the readily available consoles and flashes etc, and disable the ore silo. Most other non-sec options are limited (and there's only one ion gun and smart silicons will remove it before they go loud), won't stop more than one cyborg at a time (which you really should expect if the problem is bad enough you need to blow the borgs), and more to the point, cyborgs can self-heal, are only stunned by a very limited selection of items, and have several tools to combat you indirectly like draining the air and boltshocking doors.
> 
> Yeah borgs are easy to stop if you're fighting them one at a time and they haven't so much as done a single bit of prep, _but that's never the case, is it?_
> 
> Hacked borgs on AI malf are identical to emagged borgs so you'll definitely be giving malf a fantastic buff.

Everything of what you just said is all chance. The ion rifle and ion carbine affect every silicon and machine-oriented device in the range of the blast. Even if a borg suddenly decided to brave the shots they would be knocked out from a single blast and likely killed from the other officers that would no doubt be nearby with lethals or batons that can easily wipe out borgs. Borgs if they are emagged at all or not, will do these things or try to in a stealthy manner. But how will you manage it when there are so many different ways to do it? Its all by a rushed chaotic mess that they normally try to do at the last second when they have already been found out. 

Furthermore borgs even going after RD console/Ore silo is dependent on the AI and if they are malf. This PR affects emagged borgs ONLY which means if the AI is not malf there is a great chance a good portion of borgs are in fact not emagged but only a smaller portion and the rest are subverted by laws.

And borgs have to fight in groups as is, because its a good strategy. Assistants fight in pairs, sec fights in pairs. Because fighting anyone that you know that could have better gear such as vampire/lings/traitors all come with the chance of you simply getting gutted by their weapons anyways. Borgs is a flash and smash scenario. beat them with a bat to keep distance, do enough damage to one they cant repair.

>and more to the point, cyborgs can self-heal, are only stunned by a very limited selection of items, and have several tools to combat you indirectly like draining the air and boltshocking doors.
1. Only two borgs can self heal, engineering and miner. Both of them have limited welder uses before having to refuel and likely will be unable to repair borgs unless they are in a safe position.
2. Borgs cannot drain the air as they would be hard bwoinked for it. Same with altering atmos and fucking with the engine.
3. borgs whether emagged or not will bolt and shock doors either way. Its again wholly dependent on laws and what they have

Malf AI always has to be careful when and where it emags borgs. It usually only happens when they have above all counts to go loud. Very rarely is it before.
Prep time for borg combat is literally just a toolbox or mining equipment. Thats right a miners gear could easily fuck over a borg. Probably better than sec gear can. 

Your argument for why this PR is bad is rather biased for the idea that you hold borgs in the idea that they can 1v6 sec as a medical borg or any other model that isn't sec. They cant borgs are naturally weaker in every regard to every person on the station. Even other borgs and AI. Emagged borgs already have the round removal button from the RD's office and they really don't. They are no longer station property once emagged and should not show up on the console.

Everything stated here, in your argument affects mainly non-emagged borgs which this PR does not go over, it's over Emagged ones and how they are actually not even station borgs but syndicate ones. Emagging makes you virtually a syndicate cyborg with station modules.

I understand you feel strongly about it @YakumoChen but I will tell you everything you have brought up a one crewed borg can do all the same and still be locked down and blown. And you should never blow a cyborg anyways, lock them down if you suspect they are rogue or know they are. Find them and who they are with take them back and or reset their laws. Its simple. Easy, effective for any non-emagged borg. But we don't live in a perfect little game where people aren't itchy button pressers to kill everything that exists.

**Comment by YakumoChen** [comment]
Malf hacked borgs _still behave the same as  emagged borgs_. You need to check the code on this because your perception of it is wrong.

**Comment by Echopsychotic** [comment]
Malf hacked borgs are emagged borgs when emagged by the AI. Borgs working with the traitor which also again rarely happens when they only want the borgs to either kill someone or get something. Otherwise the idea is that you lock down the borg/blow them and everything is fine and dandy, congrats you round removed 3-6 players from the round that everyone knows will never be brought back because the request to print more posi's has been denied and all others have been left as a cube to do nothing but stay in robotics or be in a pocket.

And I consider borgs underpowered because I play them. How often do you play a borg? How often do you see the amount of patients required or needed to just be emagged and know for a damn fact that if you get it to early you are damned to a 10k powercell or hope to god that your master is a robo or scientist. Otherwise, you are stuck with full knowledge your battery will run out. Its not about when in the end its about where. 

Unlike crew who can still fight with no food and can recharge stamina on the spot. Borgs don't get that luxury. IONs even deplete the battery further! and adds brute and burn damage which if they do get burn damage it cant be repaired unless their battery is removed.

Borgs that work together however is the strategy all the time, they are supposed to team up and be creative. Sec is just as creative. they have the several best options of either saying "BORGS ROGUE" and dying, or pulling out their flash. Hitting them both and sprinting for their life. Borgs would do the same when out numbered. You don't stay and fight something you are clearly outnumbered in. But sec is never without help when they are being killed. Some dent brained assistant/med/cargo/engi in any case will come around the corner after hearing that and bash your brains in with their quick fists of denting usually from their stuff they collected or had on them. 

You seem to fail to notice the superior numbers and idea of the crew when borgs have to resort to tricks and fast acting attacks all the time. law changed borgs are just the same with this.

Changing borg laws always isn't hard. it is likely the easiest thing in the game save for making your trusty mallet to keep your next shitsec away from you. It only takes some time to research it and print the board. If you print it, just purge and put on the board you need. Not hard, not difficult its just tedious and you make it sound like some big problem. Its not, if you cant deal with the simple console then why bother doing it and just get your trusty mallet or flash or even laser pointer. It will all end in the same way with your borg going dark and them getting beaten into crit where they cant heal themselves out of it unless there is an engineering borg also subverted/emagged and that is a gamble since as a emagged borg you have no idea who is what on your screen.

So you said borgs are lethal in groups. Thats true, but everyone is. Borgs weapon is outsmarting and jumping their enemy. Using all access and their tech that may or may not help them to evade or take down. 
Sec is just as lethal, anyone is just as lethal. A flash does not stop the train of sec and assistants that already have their mighty toolboxes and crowbars ready to bash and break your chassis. 
Even now with the labsec that is being test merged sec already has their mobile lethals that they can hack themselves so 
> flash
> lethal laser
> flash 
> repeat
Run out of laser just use your trusty melee stun button to put that borg in crit or death where they can only be repaired by the robo or if they are emagged get taken apart of locked down for 20 min through cutting many of the various wires they have.

I fail to see how your point says borgs are OP if anything it again states that borgs have to use their brain which a lot of people don't to fight them. They are machines and if I recall correctly from @miniusAreas 
>Cyborgs are very powerful tools
its what they are and how they are meant to be when used by a traitor, even with a brain they are considered tools and if one dies the traitor will never EVER fix them. Borgs are tools to the traitor/ling/vampire/etc.
Saying they are anything else is an insult to the antag that forced them to do it. Borgs are powerful because the player that controls them makes them that way, not the tools. A borg that doesn't think of every action he makes is gonna fuck up and get killed easy peasy.

To summarize this: Borgs are only better in one aspect. They can think ahead of the actual crew and plan better to take down their opponent. Even then the AI is the ace strategist, not the borg himself. Chain of command is a wonderful thing and it is important with borgs. as everyone always loves to spout. Law 2 a simple fact that fucks over any borg any time but when free of it and allowed to really think of a way to solve a problem. Oh god can a borg out-think sec in a heartbeat.

**Comment by SpaceVampire** [comment]
I changed my mind and now think we should merge this, because the idea of dabbing on the whole station with my cyborg army the next time I roll antag, out weights my hatred of cyborgs for the moment. I also never play security or RD so its not like the downsides are really relevant to me anyway.

**Review by Gandalf2k15 — APPROVED** [review]
Final checks complete, Unable to merge due to stalemate.

**Comment by Echopsychotic** [comment]
@Gandalf2k15 What do you mean by stalemate? In voting for it to be merged by player opinion or being merged by the coder team?

**Comment by MLGTASTICa** [comment]
@Gandalf2k15  If its a stalemate you should discuss with the host/maintainers if its a step in the direction the server wishes to be.

**Comment by Tbowes-dev** [comment]
Cyborgs shouldnt have this protection innate. They dont need it since their powerful tools already. A traitor robotics has to spend 5 TC for this feature of encrypted codes. We cant give this to the do it everything EMAG as it makes it the one stop tool to a super powerful ally. Malfs have meta shielding so they can go about their objectives and will have to suffer the consequences of going loud like everyone else and not have a Private force.

**Comment by MLGTASTICa** [comment]
Malf AI itself has many issues, it only barely ever has objectives and it can never do conversion and can very rarely reason to do the nuke. Having a button to delete all trators once they go loud doesn't exist either , and it proves no reason to roleplay with them if they can never get a standing point. The roboticist module is more of a saboteur module for any cyborg , and the scrambled_codes comes with it automatically.

**Comment by Tbowes-dev** [comment]
Malf Cyborgs arnt traitors, their just required to follow you. Like if the CMO rolled a traitor and it brainwashed the doctors to follow them. I imagine encrypted codes is something you could make into a Malf power If needed

**Comment by Commandersand** [comment]
>extremly unfair for the cyborgs

do you know what else is unfair
all access antagonists with remote access to electronics, who are also immune to conventional stun weapons

**Comment by MLGTASTICa** [comment]
cyborgs can be stunned with flashes/flashbangs and emps , thats preety conventional , especially the flashes. ontop of the fact that they are slower than anyone.

**Comment by Weblure** [comment]
I think this is a good start to build off of.

Here's my two cents:

- The option to lock down/detonate borgs should not be fully removed.
- Locking down a borg should have a time limit (think handcuffs), and a delay before you can activate it again.
- Detonating a borg should sound an audible alarm (either over intercom or from the borg itself) and have a delay of at least 30 seconds before they detonate.
- This abilities should require captain access (if they don't already).
- A borg being detonated should be able to be 'disarmed' by a human via cutting their wires, and a locked down borg can be unlocked early the same way.

**Comment by MLGTASTICa** [comment]
@Weblure  This PR doesn't fully remove the feature , it just makes you unable to affect the hacked/emagged ones.  Leaving any kind of lockdown in any way would just make it unfun , imagine being locked down while being chased , and there's nothing you can do about it , you're getting flashed and stunned. Leaving the detonation in with a delay would be even worse , imagine if you're a malf AI and literaly have no engineering cyborgs , all your cyborgs would be ticking bombs ,which is just slightly less worse than them blowing instantly , i don't even think other cyborgs can DEFUSE eachother since the gripper might not be able to pull out the battery to start cutting wires. Making it delayed would only have people constantly spamming the detonation at the console and waiting to do it again ,so the cyborgs are once again forced to save themselves , ontop of the fact that the engineering cyborg can never save himself  , he has to be saved by another cyborg. This would mean that a malf AI would need 2 engineering cyborgs to even prevent detonations , ontop of having all the cyborgs be in the proximity of eachother + TCOMMS being online to show the warning. Malf AI rarely gets more than 3 cyborgs , and being forced to allocate 2 engineering cyborgs from what is a better fighter like medical / security is a big disadvantage.

**Comment by Jeweledbug** [comment]
My reasoning to why I don't think this merge is a good idea and should be replaced with a simpler solution.

>Emagged/Hacked cyborgs can no longer be remotely detonated/locked

This is the PR in question.

>Being able to perma-kill any malfunctioning / emagged cyborg spontaneously and without warning is both extremly annoying and extremely unfair for the cyborgs . You can't remotely detonate a traitor , so you shouldn't be able to remotely detonate a hacked cyborg either.

This is the reasoning in question. 

The first issue with it is the reasoning done so. For example, yes, you can't detonate a traitor, but emagged cyborgs are not regular traitors, the closest thing too it is a converted antagonist since the vast majority of times Cyborgs do not start malfunctioning. This to me is similar to a revolutionary, where a simple tool and one item can convert people to your cause. Or, if you have the board, you can ignore this entirely and have all cyborgs connected to a hacked AI instantly converted. It is not comparable to a regular traitor when a single antagonist can convert 

In this list (Not including S.A.M)
https://imgur.com/a/FNzhqpw
There is 10 silicons in the round, including the AI. In a round where there were there were 90 crew on the manifest. This means that a VERY significant percentage of the total players can be easily converted towards the goals of a single antagonist.

My second reason is that this will inevitably shift peoples behavior towards rogue AI/Borgs. People are trigger happy as it is with rogue silicons. With this it will mean they will jump to the gun at the first sight and simply rush the AI/Borgs guns blazing rather than even have the OPTION of locking them down. 

Solution: The solution is to make it harder than just getting the RND to rush the console and to lessen the damage of the explosion. Have it require the cards of two heads similar to a red alert call. And to have the explosion of emaggged/rogue silicons smaller and without the removal of the MMI/Pos brain. This way even if they're caught and exploded it won't remove them from the round and they can still enjoy regular play without having to remove the idea of subtly all together.

**Comment by MLGTASTICa** [comment]
@Jeweledbug  Rounds in where there are 10 Cyborgs are extremly rare , and even if the AI gets law 0'ed it doesn't provide any form of detonation immunity to all the other cyborgs.  The traitor would have to waste a emag charge for each cyborg he'd wish to give immunity to. Making the consoles harder to obtain would still allow for people to pre-prep and print them instantly ,. Its much better to not force Malf AI's to commit max saboutage AND CRIPPLE the entire station in their hunt to not have their cyborgs detonated than keep the feature in and have traitors not emag cyborgs out of fear of them being ticking bombs.

**Comment by JGlitch** [comment]
Well maybe emagging a borg could be the loud way to gain control over a borg and subverting a borg could be the sneaky way of doing it. Emagging would display a radio alert in the department channel the borg's module is coming out of the borg itself, and they would be inmune to the robotics console at that point.
But you know, people know that borg is bad now.
Subverted borgs can still be locked down if suspected, tho, or even blown up if that's the only option.

**Comment by Gandalf2k15** [comment]
It's not sensible to force a bunch of people to be effectively locked down and/or round removed the moment they do something minorly not asimov and someone manages to scream out about it just because people can and will print and keep a board in their back pocket for that very potential occasion. Knowing that before the storm, the borgs will obviously have to attempt to remove the boards and consoles, and should they succeed; unfortunately for them, people can build. And this bug is a thing.

They already trade run speed, as cyborgs. Most modules aside of security are already effectively useless even when emagged. They are trading the subtlety of being able to state laws for essentially nothing due to it. Any assistant can kill them with a flash.
They don't need the ability to click a button in some nondescript remote maintenance tunnel where they've managed to fashion a robotics console and blow every emagged borg up.

Quote from Checkhov.

**Comment by ThatGuyFromThatPlace** [comment]
Why not just add a traitor item that allows you the option of making them immune? That way the traitor has to actively sacrifice their other gear to make their borgs powerful, instead of allowing them an army of borgs as well as enough TC for whatever else they want. Same with malf AI. Make them sacrifice something for it.

Hell, you could just add a number of traitor roboticist unique items; But, the point is, something this powerful shouldn't be innate; That being the ability to create an army of all access slaves that will help you no matter what and are immune to being stopped via anything other than combat, and cannot be roleplayed or reasoned with as, their laws don't allow them to be reasoned with or persuaded, and, assuming the borg doesn't simply bolt/shock doors and stop combat from ever starting, especially engi-borgs

**Comment by MLGTASTICa** [comment]
@ThatGuyFromThatPlace  Cyborgs by themselves are not that powerfull , and no point in forcing traitors to spend  more TC , considering that a emag costs 6 TC ,which is around 30% Of their total budget.

**Comment by hugmeorelse** [comment]
I appreciate how relatively civil the conversation for this PR has been. It's quite a refreshing change of pace.

**Comment by hugmeorelse** [comment]
As host, I appreciate how relatively civil the conversation for this PR has been. It's quite a refreshing change of pace. I hope to see more exchanges like these in the future.

**Comment by YakumoChen** [comment]
This is still a very buggered idea. I don't see how this improves "round removal". Cyborgs are unlikely to rebuilt anyway if they are killed and removing the detonate option does not leave any incentive to not kill rogue borgs (and in fact just encourages it)

**Comment by MLGTASTICa** [comment]
@YakumoChen  The cyborgs can be detonated because we are on TG Code where rounds don't last long
This improves a lot for the cyborg , since now they can't be round removed instantly and without warning. 
Player voting leads to better gameplay sometimes , this one applies , it is a balance change and even if the playerbase is divided on it , theres slightly more people wanting it than people that don't (Speaking currently when i looked)
Cyborgs are no longer to self-sacrifice and serve the crew when they are emagged , they become the tools of the traitor that has emagged them .
Cyborgs even in high numbers cannot steamroll , and anyone can steamroll in big armies , cyborgs is not one of those mobs, a EMP can take out as many cyborgs as there are in its vicinity, and flashes can and will stunlock any cyborg regardless , steamrolling is usually ineffective .

**Comment by Somememeguy** [comment]
You can still lock-down a cyborg if you flash stun them and bring them to a roboticist, even without a robotics console. It'll give the robotics a job to do.

**Comment by YakumoChen** [comment]
EMPs are not widely available (theres only one gun for it and if a borg removes it or its lost you're fucked) withiut research and the assumptions that the gun is printable at the time. Security should not rely on it at all.

Relying on the robo is even worse since the flash can burn out, they might not be available at all, there might be too many borgs to keep locked down, or the robo might be the actual traitor snagging them and obviously isnt willing. 

Killing rogue borgs is always the best option in the absence of a robotics console or the means to use it, and thats no better than blowing them.

Steam rolling is quite effective, especially if the borgs are backed up by the antag in question to counteract common means of dealing with borgs.

Borgs SHOULD be self sacrifice. Its in their laws to do so.

**Comment by Gandalf2k15** [comment]
We will be test merging this to see how it goes, this was decided after in-depth discussion.

**Comment by Weblure** [comment]
@MLGTASTICa 

> This PR doesn't fully remove the feature , it just makes you unable to affect the hacked/emagged ones.

It pretty much does, since nobody's going to be blowing up/locking down borgs that aren't malf without getting bwoinked.

> Leaving any kind of lockdown in any way would just make it unfun , imagine being locked down while being chased , and there's nothing you can do about it , you're getting flashed and stunned.

There are plenty of 'unfun' things in this game that are an important part of balance. This is a compromise solution. If you get locked down while being chased, then that's good coordination on the crew's part. It certainly beats being locked down in space or maint and then being stuck there unable to do literally anything for upwards of an hour until someone finds you.

>Leaving the detonation in with a delay would be even worse , imagine if you're a malf AI and literaly have no engineering cyborgs , all your cyborgs would be ticking bombs ,which is just slightly less worse than them blowing instantly , i don't even think other cyborgs can DEFUSE eachother since the gripper might not be able to pull out the battery to start cutting wires. Making it delayed would only have people constantly spamming the detonation at the console and waiting to do it again ,so the cyborgs are once again forced to save themselves , ontop of the fact that the engineering cyborg can never save himself , he has to be saved by another cyborg. This would mean that a malf AI would need 2 engineering cyborgs to even prevent detonations , ontop of having all the cyborgs be in the proximity of eachother + TCOMMS being online to show the warning. Malf AI rarely gets more than 3 cyborgs , and being forced to allocate 2 engineering cyborgs from what is a better fighter like medical / security is a big disadvantage.

You're missing the point. Again, this is a compromise. The console would have a cooldown on these functions, as stated, so that they cannot be spammed. Even if you can't get someone to diffuse you, you can at least move to a better area so that you aren't instantly blown up in space without warning and permanently removed from the round. Pretending like the AI will be forced to have 2 engiborgs just because they now have the *option* to defuse is a ridiculous argument. They don't *have* to. They could easily choose to have no engi borgs and not be any worse off than they are now. The timer isn't solely to give time to defuse, anyway. Use your imagination. 

It's clear this vote isn't going to win in either direction, so a compromise *has* to be made. Something toned-down like this would be a good step in a better direction and give everyone a chance to test the waters, to see if not having the option to instantly detonate borgs is really that bad. The alternative is that we all be stubborn and nothing gets changed.

**Comment by MLGTASTICa** [comment]
@Weblure  I honestly won't compromise on this , its either all in or all out . LEAVING any sort of ability to lockdown / detonate emagged cyborgs will just be stressfull, and the capability to detonate emagged/hacked cyborgs is not a important part of balance , cyborgs are already very weak.
Allowing people to be remotely round-removed without any viable counter is just poor balance.

**Comment by YakumoChen** [comment]
Have you considered cyborgs _should_ be weak?

**Comment by MLGTASTICa** [comment]
@YakumoChen  cyborgs are already VERY WEAK by themselves.

**Comment by GrayRachnid** [comment]
A good malf AI with competent borgs will take out any and all possible methods to blow cyborgs. If someone pockets a board before the AI was made a threat then that's just powergaming/metagaming on their account.

The robotics console should be a threat to a malf AI or subverted cyborgs.

**Comment by Tename** [comment]
> A good malf AI with competent borgs will take out any and all possible methods to blow cyborgs. If someone pockets a board before the AI was made a threat then that's just powergaming/metagaming on their account.
> 
> The robotics console should be a threat to a malf AI or subverted cyborgs.

Ya, science can just spit out more.

**Comment by YakumoChen** [comment]
@MLGTASTICa they aren't. Aside from speed debuffs, Borgs have no slowdown, stun resistance, fire immunity, all access, secret comms, and a lot of powers to entrap and ensnare. Decently played borgs are very powerful and only security and science have the proper tools to deal with them, both of which silicons can heavily sabotage well before they go loud.

I feel like you've never played against a decently threatening borg force because it's not something you can just throw a spear at. Emagged borgs gain access to numerous and better stuns on top of this and when played as a team (silicons are a team role) can decimate shit easily.

**Comment by MLGTASTICa** [comment]
@YakumoChen as a counter side to them having normal stun immunity , no slowdown caused by leg damage , and fire immunity + AA +  comms  , they have no way to counter ions / flashes / flashbangs . This will quickly decimate any cyborg as you can just spam 6 ions to completly ruin them,  and all of these are HARDSTUNS , meaning they cannot move , cannot see , and cannot do anything if you get a hit on them , they are completly helpless and will likely be destroyed if a stun is even landed.

**Comment by YakumoChen** [comment]
> 
> 
> @YakumoChen as a counter side to them having normal stun immunity , no slowdown caused by leg damage , and fire immunity + AA + comms , they have no way to counter ions / flashes / flashbangs . This will quickly decimate any cyborg as you can just spam 6 ions to completly ruin them, and all of these are HARDSTUNS , meaning they cannot move , cannot see , and cannot do anything if you get a hit on them , they are completly helpless and will likely be destroyed if a stun is even landed.

they have very, very specific weak points that rely on a player having:
-Access to the counter item
-Using it before the cyborg kills you

Having a few counter items is like saying "humans are weak to stun batons". Yes, fucking _obviously_ but their strength is not entirely defined by a few limited-access hard stuns, that's simply a natural balance consequence of having a large amount of extremely overpowered abilities. If you think cyborgs are so weak why not just remove the hard stuns and robotics console entirely? 

They aren't weak, they're a powerful ally with different weaknesses than humans, and _extremely powerful_ utility in return.

**Comment by Weblure** [comment]
>Having a few counter items is like saying "humans are weak to stun batons". Yes, fucking obviously but their strength is not entirely defined by a few limited-access hard stuns, that's simply a natural balance consequence of having a large amount of extremely overpowered abilities.

Humans have a large amount of extremely overpowered abilities via being able to create and use items, as well has having much larger numbers.

> -Hoping the cyborg can't just dodge

A human can dodge much more easily than a borg can...

> -Using it before the cyborg kills you

Yeah, that's kinda how all combat works in SS13, lmao. You try to use an item on a target faster and more times than the target can use an item on you.

> -Access to the counter item (flashes are the only one anyone who isn't security even has a real chance of acquiring, and that requires going into melee with a cyborg)

Flashes are dirt-easy to get a hold of, and they're an instant-win item. Not even stunbatons on a person are nearly as strong as a flash on a borg. Just one click on the massive, slow borg sprite is all it takes, and then everyone within earshot will charge over and beat the shit out of you while you're kept in the hardest stunlock possible in this game.

>-Getting the item in a way before the cyborg destroys/limits the item (common with ion, which is the most powerful option)

You're not _required_ to have these items to bring down borgs, just like you aren't required to have a stun baton to bring down a human. Enough hits with just about anything will bring the borg down, and you'll always be able to outrun it. Don't wanna die from a borg? Just run away.

Literally all these points can be said for humans too. Does this mean we should have a killswitch for crew as well?

**Comment by GrayRachnid** [comment]
> 
> 
> > A good malf AI with competent borgs will take out any and all possible methods to blow cyborgs. If someone pockets a board before the AI was made a threat then that's just powergaming/metagaming on their account.
> > The robotics console should be a threat to a malf AI or subverted cyborgs.
> 
> Ya, science can just spit out more.

Not if you take out any and all methods for them to spit out more, of which there are only 3 on the station. Take out the console in the RD's office and the board in secure and now nobody can blow you up.

**Comment by MLGTASTICa** [comment]
@GrayRachnid  this is  a TG codebase ,  the robotics control console is printable from 1)Robotics 2)Science 3)Engineering , all these imprinters have to be blown up , followed by 4)Engi techfab 5)Research techbag , along with the removal of the initial boards.
This is extremly hard to achieve without being spotted , since the offices are placed in public locations and cyborgs will very easily be spotted , and people do tend to meta when they see a blue APC , building a robotics control console for no reason other than having it ready incase AI goes roque.

**Comment by Jungling** [comment]
Locking down emagged borgs is fine. Detonating would make no sense since you've emagged the electronic it shouldn't be effected by outside interferance thats lethal anyway. Playing cat and mouse with lockdown monitors is a bit of a chessboard for emagged borg. I think removing BOTH it outright is a massive buff to emagged borgs... Don't want that.  However removing simply detonation is a great idea! I don't like murderboning and anything that helps fix it is definately good!
